.github/workflows/build.yaml

```diff
@@ -0,0 +1,30 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+      - 'release/**'
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    name: Build on multiple version of macOS
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - macos-13
+          - macos-14
+          - macos-15
+    runs-on: ${{ matrix.platform }}
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+        with:
+          fetch-depth: 1
+      - name: Build
+        run: make
```

Comment:
`make test.repro` should be tested

Reply:
I think I copied it from #63. You want to replace with `make` with `make test.repro`?

Reply:
The release workflow run `make test.repro` why test it here again? If we want to test this on all architectures we can add more platforms to the release matrix.